### DISCUSSION TITLE
The latest kong version (2.8.0) returns an error on GO plugin integration ,  Not a plugin info table:

### DISCUSSION THREAD

**Discussion by liuqiyu-community** [Kong/kong#8685] [Help]
### Summary
The latest kong version (2.8.0) returns an error on GO plugin integration.

### Steps To Reproduce
1). download https://github.com/liuqiyu-community/go-kong-plugins.git
2)cd test
3)docker-compose up

### Additional Details & Logs

2022/04/14 04:46:47 [warn] 1#0: the "user" directive makes sense only if the master process runs with super-user privileges, ignored in /usr/local/kong/nginx.conf:6
nginx: [warn] the "user" directive makes sense only if the master process runs with super-user privileges, ignored in /usr/local/kong/nginx.conf:6
2022/04/14 04:46:47 [debug] 1#0: [lua] globalpatches.lua:10: installing the globalpatches
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:460: init(): [dns-client] (re)configuring dns client
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:465: init(): [dns-client] staleTtl = 4
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:469: init(): [dns-client] noSynchronisation = nil
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:488: init(): [dns-client] query order = LAST, SRV, A, AAAA, CNAME
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:528: init(): [dns-client] adding A-record from 'hosts' file: a67b57f9a05f = 172.21.0.2
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:528: init(): [dns-client] adding A-record from 'hosts' file: localhost = 127.0.0.1
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: localhost = [::1]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-mcastprefix = [ff00::0]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-localhost = [::1]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-loopback = [::1]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-allrouters = [ff02::2]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-localnet = [fe00::0]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-allnodes = [ff02::1]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:552: init(): [dns-client] validTtl = nil
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:593: init(): [dns-client] nameserver 127.0.0.11
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:598: init(): [dns-client] attempts = 5
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:605: init(): [dns-client] no_random = true
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:614: init(): [dns-client] timeout = 2000 ms
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:618: init(): [dns-client] ndots = 0
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:620: init(): [dns-client] search = 
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:633: init(): [dns-client] badTtl = 1 s
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:635: init(): [dns-client] emptyTtl = 30 s
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:460: init(): [dns-client] (re)configuring dns client
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:465: init(): [dns-client] staleTtl = 4
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:469: init(): [dns-client] noSynchronisation = false
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:488: init(): [dns-client] query order = LAST, SRV, A, CNAME
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:528: init(): [dns-client] adding A-record from 'hosts' file: a67b57f9a05f = 172.21.0.2
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:528: init(): [dns-client] adding A-record from 'hosts' file: localhost = 127.0.0.1
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: localhost = [::1]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-mcastprefix = [ff00::0]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-localhost = [::1]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-loopback = [::1]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-allrouters = [ff02::2]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-localnet = [fe00::0]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-allnodes = [ff02::1]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:552: init(): [dns-client] validTtl = nil
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:593: init(): [dns-client] nameserver 127.0.0.11
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:598: init(): [dns-client] attempts = 5
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:605: init(): [dns-client] no_random = true
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:614: init(): [dns-client] timeout = 2000 ms
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:618: init(): [dns-client] ndots = 0
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:620: init(): [dns-client] search = 
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:633: init(): [dns-client] badTtl = 1 s
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:635: init(): [dns-client] emptyTtl = 30 s
2022/04/14 04:46:47 [debug] 1#0: [lua] globalpatches.lua:270: randomseed(): seeding PRNG from OpenSSL RAND_bytes()
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:460: init(): [dns-client] (re)configuring dns client
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:465: init(): [dns-client] staleTtl = 4
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:469: init(): [dns-client] noSynchronisation = false
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:488: init(): [dns-client] query order = LAST, SRV, A, CNAME
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:528: init(): [dns-client] adding A-record from 'hosts' file: a67b57f9a05f = 172.21.0.2
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:528: init(): [dns-client] adding A-record from 'hosts' file: localhost = 127.0.0.1
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: localhost = [::1]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-mcastprefix = [ff00::0]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-localhost = [::1]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-loopback = [::1]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-allrouters = [ff02::2]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-localnet = [fe00::0]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:543: init(): [dns-client] adding AAAA-record from 'hosts' file: ip6-allnodes = [ff02::1]
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:552: init(): [dns-client] validTtl = nil
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:593: init(): [dns-client] nameserver 127.0.0.11
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:598: init(): [dns-client] attempts = 5
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:605: init(): [dns-client] no_random = true
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:614: init(): [dns-client] timeout = 2000 ms
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:618: init(): [dns-client] ndots = 0
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:620: init(): [dns-client] search = 
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:633: init(): [dns-client] badTtl = 1 s
2022/04/14 04:46:47 [debug] 1#0: [lua] client.lua:635: init(): [dns-client] emptyTtl = 30 s
2022/04/14 04:46:47 [debug] 1#0: [kong] process.lua:62 search config for pluginserver named: hello
sh: /usr/local/bin/hello: not found
2022/04/14 04:46:47 [error] 1#0: init_by_lua error: ...al/share/lua/5.1/kong/runloop/plugin_servers/process.lua:160: Not a plugin info table: 
/usr/local/bin/hello -dump

stack traceback:
	[C]: in function 'error'
	...al/share/lua/5.1/kong/runloop/plugin_servers/process.lua:160: in function 'ask_info'
	...al/share/lua/5.1/kong/runloop/plugin_servers/process.lua:197: in function 'get_plugin_info'
	...local/share/lua/5.1/kong/runloop/plugin_servers/init.lua:284: in function 'get_plugin'
	...local/share/lua/5.1/kong/runloop/plugin_servers/init.lua:292: in function 'load_plugin'
	/usr/local/share/lua/5.1/kong/db/dao/plugins.lua:136: in function 'load_plugin_handler'
	/usr/local/share/lua/5.1/kong/db/dao/plugins.lua:221: in function 'load_plugin'
	/usr/local/share/lua/5.1/kong/db/dao/plugins.lua:273: in function 'load_plugin_schemas'
	/usr/local/share/lua/5.1/kong/init.lua:543: in function 'init'
	init_by_lua:3: in main chunk
nginx: [error] init_by_lua error: ...al/share/lua/5.1/kong/runloop/plugin_servers/process.lua:160: Not a plugin info table: 
/usr/local/bin/hello -dump

stack traceback:
	[C]: in function 'error'
	...al/share/lua/5.1/kong/runloop/plugin_servers/process.lua:160: in function 'ask_info'
	...al/share/lua/5.1/kong/runloop/plugin_servers/process.lua:197: in function 'get_plugin_info'
	...local/share/lua/5.1/kong/runloop/plugin_servers/init.lua:284: in function 'get_plugin'
	...local/share/lua/5.1/kong/runloop/plugin_servers/init.lua:292: in function 'load_plugin'
	/usr/local/share/lua/5.1/kong/db/dao/plugins.lua:136: in function 'load_plugin_handler'
	/usr/local/share/lua/5.1/kong/db/dao/plugins.lua:221: in function 'load_plugin'
	/usr/local/share/lua/5.1/kong/db/dao/plugins.lua:273: in function 'load_plugin_schemas'
	/usr/local/share/lua/5.1/kong/init.lua:543: in function 'init'
	init_by_lua:3: in main chunk

**Comment by StarlightIbuki** [comment]
It says the Nginx cannot find `/usr/local/bin/hello`. It looks like you copy the plugin executable to the path in the Docker file. You can check your image whether that file is present.
